metadata-db-app/src/config/migrate_config.clj

```diff
@@ -1,5 +1,6 @@
 (ns config.migrate-config
   "Provides the configuration for Drift migrations."
```

Comment:
I don't think these namespaces are deprecated, they are still used when running Lein commands. We should add more info to the docstring to explain when these namespaces are used. Something like when running `lein create-user/drop-user/db-migrate` this namespace is used. The database migration endpoints use a different namespace. The reason we need both namespaces is because the lein commands require ...(? Leo seems to know the answer ?), but when we run services from the dev-system uberjar we need to ensure namespaces do not conflict across applications.

Reply:
Done.